app/javascript/components/Card.jsx

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import Button from 'components/Button';
 
 import 'stylesheets/card';
 
-const Card = ({ children, className }) => (
-    <div className={`card p-10 ${className}`}>{children}</div>
+const Card = ({ children, className, link }) => (
+    <>
+        <div className={`card p-10 ${className}`}>
+            {children}
+            <div className="flex flex-row justify-between items-center space-x-5">
+                <Button type="white" className="w-20">
+                    <a href={link} target="_blank" rel="noopener noreferrer">
+                        Apply
+                    </a>
+                </Button>
+            </div>
+        </div>
+    </>
```

Comment:
Because Card is a component that can be found on several pages. This code has an effect on them in some way. Let me show you a list derived from VsCode:
  <img width="224" alt="Screen Shot 2023-02-07 at 11 29 01 PM" src="https://user-images.githubusercontent.com/26725925/217433753-37be2149-243c-4dde-956d-985acbff9fe7.png">

This code has impacted another Card element unrelated to Jobs on the same page:

   <img width="727" alt="Screen Shot 2023-02-07 at 11 31 09 PM" src="https://user-images.githubusercontent.com/26725925/217434036-ef405da3-1010-4c7d-8f8b-b9153e3e1f55.png">

Components are often designed to maintain consistent styling across several pages, which is why they may be so plain.

Reply:
@luciagirasoles Thank you for the feedback. I forgot to check if the Card component appeared elsewhere in the page. I will fix my code.